omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -25,7 +43,25 @@ else
     PREFIX="/usr"
 fi
 
-binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client"
+binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
+
+old_hab_path=$(which hab)
+
+if [ "$old_hab_path" != "" ]; then
+    loc=$(readlink "$old_hab_path") # If it's not a link readlink gives back the same path.
+    case $loc in
+      *chef-workstation*)
+        ;;
+      *)
+        echo ""
+        echo "NOTE: Chef Habitat on this node is now managed by Chef Workstation."
+        echo "      Future Workstation updates will always include the latest stable"
+        echo "      Habitat release."
+        echo ""
+        ;;
+    esac
+    rm -rf "$old_hab_path"
```

Comment:
I'm conflicted on this one. It would surprise me if I installed Visual Studio Code and that decided to uninstall notepad++ for me. I realize that we own both Chef Workstation and Habitat but it still seems weird...

Reply:
Updating for a note made during manual review - I am in favor of overwriting an existing symlink in `$PREFIX/bin` and renaming an existing binary from `$PREFIX/bin/hab` to `$PREFIX/bin/hab.old`